lib/screens/dup_screen_data/response.ex

```diff
@@ -60,10 +60,18 @@ defmodule Screens.DupScreenData.Response do
     end
   end
 
+  defp partial_alert_specifier(%{headsign: {:adj, "Ashmont/Braintree" = headsign}}) do
```

Comment:
Seems like this signature only ever matches Kenmore "westbound" (via quick search for the `:adj` tag. So I don't think this is needed.